Soccar [Client]/src/nl/soccar/ui/fx/models/MapUiFx.java

```diff
@@ -39,40 +41,56 @@
      */
     public MapUiFx(GameCanvasFx canvas, Map model) {
         super(canvas, model);
-
-        addWalls();
     }
 
-    private void addWalls() {
+    public void addWalls() {

```

Comment:
Moet deze methode persé public zijn? Ik vraag me af of hier een andere oplossing voor is.